src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.html

```diff
@@ -60,8 +60,8 @@ <h3 class="font-bolt order-title">{{ 'order-details.your-order' | translate }}</
             <p class="order-header">{{ 'order-details.store-delivery' | translate }}</p>
             <ul class="order-list d-flex">
               <li class="order-list-item">{{ 'order-details.order-numbers' | translate }} &nbsp;</li>
-              <li class="order-list-item" *ngFor="let order of additionalOrders">
-                <strong>{{ order }}, &nbsp;</strong>
+              <li class="order-list-item" *ngFor="let order of additionalOrders; let i = index">
+                <strong>{{ order }}{{ i < additionalOrders.length - 1 ? ',&nbsp;' : '' }}</strong>
```

Comment:
We can change this to 'let last = last'
 <strong>{{ order }}{{ !last ? ',&nbsp;' : '' }}</strong>
https://angular.io/api/common/NgForOf#local-variables

Reply:
done